python/test/t_LeastSquaresExpansion_std.py

```diff
@@ -0,0 +1,116 @@
+#! /usr/bin/env python
```

Comment:
The reference `python/test/t_LeastSquaresExpansion_std.expout` is missing in the PR.

Reply:
The __str__ and __repr__ functions are called but the output is not checked (see the IGNOREOUT option in python/test/CMakeList and lib/test/CMakeList), only the numerical results are checked using assertions.